lib/evse_security/evse_security.cpp

```diff
@@ -635,6 +640,36 @@ OCSPRequestDataList EvseSecurity::get_ocsp_request_data() {
     return response;
 }
 
+OCSPRequestDataList EvseSecurity::get_ocsp_request_data(const std::string& certificate_chain,
+                                                        const CaCertificateType certificate_type) {
+    std::lock_guard<std::mutex> guard(EvseSecurity::security_mutex);
+
+    OCSPRequestDataList response;
+    std::vector<OCSPRequestData> ocsp_request_data_list;
+
+    X509CertificateBundle leaf_bundle(certificate_chain, EncodingFormat::PEM);
```

Comment:
I'd also wrap this up in a try/catch (see https://github.com/EVerest/libevse-security/blob/7ad7839125a61e159eca04b2efbbd638039cdc4f/lib/evse_security/evse_security.cpp#L626) because we never know when a bundle can be malformed.